08-graphs/02-graphs-algorithms.md

```diff
@@ -10,18 +10,6 @@ In breadth-first-search we start with a particular node and visit each node conn
 
 We do so by adding each of the neighbors of the starting node to a Queue and then loop through the Queue removing an element and repeating the process with the neighbors of the current node.
```

Comment:
```suggestion
We do so by adding each of the neighbors of the starting node to a queue. Then we loop through the queue, removing the first element in the queue and adding each of its unvisited neighbors to the queue. We repeat this process until the queue is empty. 

Once the queue is empty we know we have visited all nodes in the graph that are possible to reach from the starting node.

Reply:
I think it may be helpful to have a video walkthrough of what walking through this 'pseudocode' looks like with an example graph before asking them to implement.